docs/Networking.asciidoc

```diff
@@ -409,21 +150,36 @@ PRE_UP_SCRIPT="wicked:gre_tunnel_preup.sh"
 
 Ensure to make gre_tunnel_preup.sh executable.
 
-Allow GRE in older setups still using SuSEfirewall2:
-
-[source,sh]
-----
-# /etc/sysconfig/SuSEfirewall2
-FW_SERVICES_EXT_IP="GRE"
-FW_SERVICES_EXT_TCP="1723"
-----
-
 NOTE: When using GRE tunnels keep in mind that virtual machines inside the ovs
 bridges have to use MTU=1458 for their physical interfaces (eth0, eth1). If
 you are using support_server/setup.pm the MTU will be set automatically to
 that value on support_server itself and it does MTU advertisement for DHCP
 clients as well.
 
+===== Configure openQA workers
+Allow worker intstances to run multi-machine jobs:
+
+[source,sh]
+----
+# /etc/openqa/workers.ini
+[global]
+WORKER_CLASS = qemu_x86_64,tap
+----
+
+NOTE: The number of tap devices should correspond to the number of the running
+worker instances. For example, if you have set up 3 tap devices, the same
+number of worker instances should be configured.
```

Comment:
Swapping the order here would be more natural to me:
```suggestion
worker instances. For example, if you have set up 3 worker instances, the same
number of tab devices should be configured.
```
Or just remove the example because the preceding sentence should be clear enough on its own:
```suggestion
worker instances.
```